### DISCUSSION TITLE
Web UI Usage questions

### DISCUSSION THREAD

**Discussion by rct** [merbanan/rtl_433#2420] [General]
I have some questions about using the web interface. There is a lot of great functionality in here. If these are documented somewhere, my apologies and please let me know.  

I think if there were a page with some screenshots in action, it would help to understand what functionality is there and how to use it.

----

What is indicated by red dots in power level graph?  I believe green dots are successful decodes.   The red dots seem to occur only occasionally, possibly when there are some collisions?

![image](https://user-images.githubusercontent.com/38770/224571702-4f510f0f-c093-4123-9b48-4b9495e97fbd.png)

![image](https://user-images.githubusercontent.com/38770/224572775-250b057a-da6e-4b01-b020-b7304cc8c657.png)

----

Is there any way to sort/resort the devices tab?

----

What needs to be enabled in order to get output in the SYS/Syslog tab?

It seems like it should show logging messages.   I'm not using any `-F syslog` output, though I am using `-F log` 

![image](https://user-images.githubusercontent.com/38770/224572219-3437b628-c27e-43c9-8b9d-298c84924dce.png)

----

What needs to be done to enable output from the protocols tab?   I never see any output so maybe something needs to be enabled?

![image](https://user-images.githubusercontent.com/38770/224571990-30d214fe-2390-4090-9256-c93728b2efc9.png)

**Comment by zuckschwerdt** [comment]
Yes, now that we can really support the HTTP API with the faster event loop some fixes and documentation will be worthwhile.

The Plot has gray: undecoded, green: OOK, red: FSK.

The other things I need to look into, they did work at some point.

**Comment by rct** [comment]
> The Plot has gray: undecoded, green: OOK, red: FSK.

Thanks -- my guesses were off, I incorrectly assumed the gray "line" was representing the noise level at the time.  Didn't realize I was seeing any FSK decodes on that device.

----

Here's an semi interesting  little quirk because of the way the Acurite Atlas has 3 different message types--The columns in the device zoom depend on which message type what last received when it was clicked to create the zoom. Of course this make sense, but was a little surprising at first So you need to wait for the message type you want that has temp/humidity, rain, or light info if you want to see the history for those.

![image](https://user-images.githubusercontent.com/38770/224576157-53539fef-5636-459a-b396-91a2571cabda.png)

![image](https://user-images.githubusercontent.com/38770/224576231-723e6068-98a5-417f-8cf8-10ba8b088c7e.png)

![image](https://user-images.githubusercontent.com/38770/224576270-723bcf63-d472-491b-907a-ff8b12de3ed6.png)

**Reply by zuckschwerdt** [reply]
No, you were right, the gray "line" is noise floor and undecoded messages would be dots. But that needs to be enabled somewhere, I think, hmm.

Good point about a need to aggregate column labels.
